web/src/main/webapp/resources/js/atsy-new-application-popup.js

```diff
@@ -0,0 +1,46 @@
+var positionDefaultValueInEng = 'Please choose a position';
+var applicationSourceDefaultValueInEng = 'Please choose an application source';
+var positionDefaultValueInHun = 'Kérem válasszon egy pozíciót';
+var applicationSourceDefaultValueInHun = 'Kérem válasszon egy jelentkezési forrást';
+
+function validateApplicationSourceAndPositionDropDowns() {
```

Comment:
Do not forget to implement this validation in the controller too.
As a rule of thumb a reminder: all validations must be on server side, client side validations are for there to improve UX and lower the request count to servers.